homework/calculate/calculate.hpp

```diff
@@ -2,6 +2,25 @@
 #include <string>
 
 std::string calculate(const std::string& command, int first, int second) {
-    // TODO: Implement your solution here and return proper value
-    return "";
+    if (command == "add") {
+        return std::to_string(first + second);
+    }
+    if (command == "subtract") {
+        return std::to_string(first - second);
+    }
+    if (command == "multiply") {
+        return std::to_string(first * second);
+    }
+    if (command == "divide") {
+        if (second != 0) {
+            return std::to_string(first / second);
+        }
+        else {
+            return "Division by 0";
+        }
+    }
+    else {
```

Comment:
When you use return inside `if` you don't need `if else` or `else`. Just write `if (sth) { return blabla; }`